src/style.css

```diff
@@ -0,0 +1,202 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  background-color: #f3f3f3;
+  color: #333;
+}
+
+ul li {
+  list-style: none;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+ul {
+  margin: 0;
+}
+
+.section-todo {
+  width: 100%;
+  min-height: 100vh;
+  padding: 120px 20px;
+}
+
+.todo-container {
+  max-width: 600px;
+  min-width: 600px;
+  background-color: #fff;
+  margin: 0 auto;
+
+  /* padding: 20px; */
+  display: flex;
+  flex-direction: column;
+  box-shadow: 1px 3px 3px #aaa;
+  border-radius: 4px;
+}
+
+.todo-title {
+  font-size: 24px;
+  font-weight: 500;
+  color: #333;
+}
+
+.todo-header,
+.add-todo-form,
+.clear-btn-wrapper,
+.todo-list-item {
+  width: 100%;
+  height: 64px;
+  padding: 14px 20px;
+  border-bottom: 1px solid #f3f3f3;
+}
+
+.todo-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.add-todo-form {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.add-input,
+.edit-input {
+  width: 100%;
+  font-size: 18px;
+  line-height: 22px;
+  font-weight: 300;
+  font-style: italic;
+  padding: 6px 12px;
+  margin-right: 4px;
+  color: #333;
+  border: none;
+  outline: none;
+}
+
+.edit-input {
+  display: none;
+}
+
+.edit {
+  display: block;
+  background-color: rgb(255, 252, 245);
+}
+
+.edit-input:hover,
+.edit-input:focus,
+.add-input:hover,
+.add-input:focus {
+  border: none;
+  outline: none;
+}
+
+#input-icon {
+  transform: rotate(90deg);
+  font-weight: 300;
+}
+
+#input-icon path {
+  fill: #333;
+}
+
+.mark-icon {
+  display: none;
+  font-size: 20px;
+  color: green;
+}
+
+.todo-list-item {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.item-editing {
+  background-color: rgb(255, 252, 245);
+}
+
+.item-info {
+  width: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.check-icon {
+  display: block;
+  color: #333;
+  cursor: pointer;
+}
+
+.todo-task {
+  display: inline-block;
+  width: 100%;
+  font-size: 18px;
+  line-height: 22px;
+  font-weight: 400;
+  color: #333;
+  margin: 0 8px;
+}
+
+.todo-task-edit {
+  display: none;
+}
+
+.action-cta {
+  display: flex;
+  align-items: center;
+}
+
+.ellipsis-icon {
+  display: block;
+  cursor: move;
+}
+
+.trush-icon {
+  display: none;
+  cursor: pointer;
+  color: rgb(180, 30, 30);
+}
+
+.show {
+  display: block;
+}
+
+.hide {
+  display: none;
+}
+
+.clear-btn-wrapper {
+  background-color: #f2f2f2;
+  border-bottom: none;
+
+  /* Layout */
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.clear-btn {
+  font-size: 18px;
+  border: none;
+  outline: none;
+}
+
+.clear-btn:hover,
+.clear-btn:focus {
+  border: none;
+  outline: none;
+  cursor: pointer;
+}
```

Comment:
border: none;
  outline: none;
These lines of code are already included in the clear-btn, please consider removing them from hover and focus options

Reply:
Hi @TamaraDimov, Thanks for your review